terraso_backend/apps/story_map/views.py

```diff
@@ -119,12 +119,17 @@ def post(self, request, **kwargs):
 
         return JsonResponse(story_map.to_dict(), status=201)
 
+def is_valid_media(media):
+    if (media and (media["type"].startswith("image") or media["type"].startswith("audio") or media["type"].startswith("video"))):
+        return True
+    logger.info("Invalid media type.")
+    raise Exception("Invalid media type")
 
 def handle_config_media(new_config, current_config, request):
     if "chapters" in new_config:
         for chapter in new_config["chapters"]:
             media = chapter.get("media")
-            if media and "contentId" in media:
+            if media and is_valid_media(media) and "contentId" in media:
```

Comment:
Issue: I think we should do this before the `handle_config_media` function gets called, right now if there are valid media before the system will upload them for S3 and then we will show the user that save failed, I think our options are:
- Omit media files that are not supported without raising an exception, maybe just logging a warning
- Validate before `handle_config_media` if there are any invalid media types and reject the request to save the story map